src/MediaPlayer/MediaPlayer.cs

```diff
@@ -303,6 +303,7 @@ public enum MPErrorCode : nint {
 		NotSupported,
 		[iOS (10,1)]
 		Cancelled,
+		RequestTimedOut,
```

Comment:
@dalexsoto no, it's an error (output) enum so it's not needed. The one on `Cancelled` should be removed (not critical).